src/functions.jl

```diff
@@ -44,14 +37,17 @@ function invpow1(b::Real, x::Real)
 end
 
 function invlog1(b::Real, x::Real)
-    if b ≥ zero(b)
-        b^x
-    else
-        throw(DomainError(x, "inverse for log($b, x) is not defined at $x"))
-    end
+    # exception may happen here: check cannot be done in inverse(f) because of log(Real, Complex)
+    b > zero(b) && b != one(b) || throw(DomainError(x, "inverse for log($b, x) is not defined at $x"))
```

Comment:
If we use `one` I guess we could directly use `isone`?
```suggestion
    b > zero(b) && !isone(b) || throw(DomainError(x, "inverse for log($b, x) is not defined at $x"))
```